packages/nvlsm/nvlsm.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Start NVIDIA Switch Manager
+RequiresMountsFor=/etc
+
+[Service]
+Type=oneshot
+ExecCondition=/usr/bin/ghostdog infiniband-present
+ExecStart=/usr/bin/nvlsm -F /etc/nvidia/nvlsm.conf -B --pid_file /var/run/nvidia-fabricmanager/nvlsm.pid
```

Comment:
Can this be made to run in the foreground, like Fabric Manager, rather than letting it do its own pid file based service management?

Reply:
I can take a look, I used the default behavior provided by the Fabric Manager start script that uses this but I think  we should consider what flags are ideal for our use. I'll see if I can get nvlsm to write to stdout.